### DISCUSSION TITLE
ESP32-S3 support??

### DISCUSSION THREAD

**Discussion by oxbown** [zauberzeug/lizard#31] [Ideas]
Is there support for ESP32-S3 boards?

**Comment by falkoschindler** [comment]
@oxbown To be honest, I don't know. We haven't worked with S3 boards yet. Would you like to try it out?

**Reply by oxbown** [reply]
I´ve got an [M5stack AtomS3 Lite](https://docs.m5stack.com/en/core/AtomS3%20Lite) , I´ll test it!

**Reply by pascalzauberzeug** [reply]
Hey @oxbown, did you have any success with your board?

**Comment by JensOgorek** [comment]
ESP32s3 Support is added with #148
